server/resources/TemplateResources.py

```diff
@@ -1,22 +1,100 @@
-from flask_restful import Resource
-from models.StepModel import StepModel
-from models.CampaignModel import CampaignModel
-from models.ProspectModel import ProspectModel
+from flask_restful import Resource, reqparse
+from models.UserModel import UserModel
 from models.EmailTemplateModel import EmailTemplateModel
-from utils.RequestParserGenerator import RequestParserGenerator
-from flask_jwt_extended import jwt_required, get_jwt_identity
+from flask_jwt_extended import (jwt_required, get_jwt_identity)
 
+parser = reqparse.RequestParser()
+parser.add_argument('name', type=str, location='json', required=True)
+parser.add_argument('type', type=str, location='json', required=True)
+parser.add_argument('subject', type=str, location='json', required=True)
+parser.add_argument('body', type=str, location='json', required=True)
```

Comment:
As for this, I made a request parser generator in utils, this can be done with:
parser = reqparsegenerator.getParser('name', .. etc) , on another note, default location is json so you don't have to specify location='json' on that parameter